aspnetcore/host-and-deploy/azure-apps/troubleshoot.md

```diff
@@ -65,12 +65,23 @@ Many startup errors don't produce useful information in the Application Event Lo
 
 1. Open **Advanced Tools** in the **Development Tools** area. Select the **Go&rarr;** button. The Kudu console opens in a new browser tab or window.
 1. Using the navigation bar at the top of the page, open **Debug console** and select **CMD**.
+
+#### Test a 32-bit (x86) app
+
 1. Open the folders to the path **site** > **wwwroot**.
 1. In the console, run the app by executing the app's assembly.
-   * If the app is a [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd), run the app's assembly with *dotnet.exe*. In the following command, substitute the name of the app's assembly for `<assembly_name>`: `dotnet .\<assembly_name>.dll`
-   * If the app is a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd), run the app's executable. In the following command, substitute the name of the app's assembly for `<assembly_name>`: `<assembly_name>.exe`
+   * If the app is a [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd), run the app's assembly with `dotnet .\{ASSEMBLY NAME}.dll`.
+   * If the app is a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd), run the app's executable with `{ASSEMBLY NAME}.exe`.
 1. The console output from the app, showing any errors, is piped to the Kudu console.
 
+#### Test a 64-bit (x64) app
+
+* If the app is a 64-bit (x64) [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd), run the app using the 64-bit *dotnet.exe* located in the `D:\Program Files\dotnet` directory:
```

Comment:
Why is this so verbose?

If the app is a 64-bit (x64) [framework-dependent deployment](/dotnet/core/deploying/#framework-dependent-deployments-fdd):

1. cd *D:\Program Files\dotnet*
1. `dotnet \home\site\wwwroot\{ASSEMBLY NAME}.dll`.
 * The console output from the app, showing any errors, is piped to the Kudu console.
 * If the app is a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd), run the app's executable with `{ASSEMBLY NAME}.exe`.

Reply:
I wouldn't characterize it that way ... I'd just call it *conversational tone*. However, one author's "conversational tone" is another author's "verbose language." :smile: lol

I'll revise it. Stand-by.

Reply:
It explains why you need to change into the directory specified. As a consumer of docs, I prefer you to tell me why I'm doing something rather than just telling me with no explanation. Up to you.

Reply:
It doesn't present clear instructions to have "cd D: ..." ... readers (e.g., newbies) could just totally choke on that.

Reply:
@jamesche75 I lean that way in just about everything I write, but I know that sometimes I get too wordy. There are some cuts that can be made here. I'll have another commit up in a few minutes, and we can take a look at a shorter version and revise further if needed.